build/tools/chainlink-fulfilling-requests.md

```diff
@@ -0,0 +1,161 @@
+# Running a Chainlink Node: Fulfilling Requests
+
+Now that we have a node [running on Fuji test network](chainlink-node-deployment.md), let's do something useful with it!
+
+With your own Oracle contract, you can use your own node to fulfill requests. This guide will show you how to deploy your own Oracle contract and add jobs to your node so that it can provide data to smart contracts.
+
+## Prerequisites
+
+Once your node is deployed, You will have to fund it with Link token. To do this, we must visit the [Fuji Chainlink Faucet](https://linkfaucet.protofire.io/fuji). Make sure you have FUJI selected as your network, enter your wallet address, and click on the **Send me 10 Test Link** button. You should receive test Link in a few seconds.
+
+We are also going to need testnet AVAX. Use the [AVAX Fuji Testnet Faucet](https://faucet.avax-test.network/) to fund the same address with testnet AVAX.
+
+### Funding your node
+
+Now that our wallet is funded, we are going to send some LINK and AVAX to our node address. You can find your Node address during deployment or in the **Keys** tab of the node operator GUI:
+
+![Node address](../../.github/chainlinkp2-01-Screen_Shot_2021-07-15_at_12.41.44_PM.png)
+
+Copy your node address and send your node LINK **and** AVAX. Using Metamask, it will look like this:
+
+![Node funding](../../.github/chainlinkp2-02-Screen_Shot_2021-07-15_at_12.44.06_PM.png)
+
+The total cost to send LINK will be LINK+gas so make sure you have both enough LINK and AVAX. Refresh the page and view your updated balances. Balances are shown in wei.
+
+![Node balance](../../.github/chainlinkp2-03-Screen_Shot_2021-07-18_at_2.27.42_PM.png)
+
+Although UI will show `ETH balance`, the balance is actually in AVAX. Your node is funded! Now it can start fulfilling requests!
+
+## Oracle deployment
+
+For this tutorial we will harmonize Chainlink contracts with the Avalanche Fuji network. First, We will need to create an Oracle to forward requests from on chain to off chain. Open [Remix](https://remix.ethereum.org/), select **Load from Gist** and paste in code from [here](https://gist.github.com/Julian-dev28/acca3b2a449f02225defc92bc2909f2f).
+
+OR
+
+Create a new file named `OracleAndConsumer.sol`:
+
+```bash
+pragma solidity ^0.6.6;
+
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/Oracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/MockOracle.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/Consumer.sol";
+import "https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.6/tests/BasicConsumer.sol";
+```
+
```

Comment:
_Note: By default Remix selects nightly Solidity build so you must change the compiler to v0.6.6_